pkg/versionstream/versionstreamrepo/gitrepo.go

```diff
@@ -238,3 +253,16 @@ func resolveRefToTag(dir string, commitish string, gitter gits.Gitter) (string,
 	}
 	return resolved, nil
 }
+
+func isBranch(dir string, ref string, gitter gits.Gitter) (bool, error) {
```

Comment:
This is useful enough that I think it would make sense to go in the git helpers.

Reply:
https://github.com/jenkins-x/jx/blob/master/pkg/gits/helpers.go specifically =)

Reply:
Done